### DISCUSSION TITLE
支持Lua文档翻译？

### DISCUSSION THREAD

**Discussion by chunzhi716** [YiiGuxing/TranslationPlugin#2544] [General]
好像目前还没有支持Lua的文档翻译，或者说这是我的问题？
![image](https://user-images.githubusercontent.com/108501102/204094061-a8430870-b7bf-429e-ab72-7127bf70162e.png)

**Comment by YiiGuxing** [comment]
Lua目前还没有IDEA官方的支持，所以其文档翻译插件暂时也没有支持：

![image](https://user-images.githubusercontent.com/10737066/204094408-11e43ae4-3bcb-4b53-bec6-9b5a1e982c98.png)
